nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/SelectHiveQL.java

```diff
@@ -113,6 +126,17 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
+    public static final PropertyDescriptor HIVEQL_POST_QUERY = new PropertyDescriptor.Builder()
+            .name("hive-post-query")
+            .displayName("HiveQL Post-Query")
+            .description("HiveQL post-query to execute. Semicolon-delimited list of queries. "
+                    + "Example: 'set tez.queue.name=default; set hive.exec.orc.split.strategy=HYBRID; set hive.exec.reducers.bytes.per.reducer=258998272'. "
+                    + "Note, the results/outputs of these queries will be suppressed if successful executed.")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
```

Comment:
Could it make sense to have a custom validator here just to be sure the statements will be accepted by the driver? (would need to check if EL is used or not first). Not sure it's worth the trouble though...

Reply:
Well, would be interesting to have a parser here, but then it should be global, and include also main query ("select" statement). I believe it can be done, but out of scope for this change. We can have another Improvement request in Jira to have new HiveQL validator for all Hive related processors.